src/riak_kv_pb_timeseries.erl

```diff
@@ -127,6 +130,53 @@ process(#tsputreq{table = Bucket, columns = _Columns, rows = Rows}, State) ->
             {reply, missing_helper_module(Bucket, BucketProps), State}
     end;
 
+%% @ignore DELETE (as if)
```

Comment:
Could you elaborate a little more in this comment? Or, maybe I'm missing something, I don't know if `@ignore` has some special meaning, but I don't understand what this line is trying to say.